dataproc/src/main/java/CreateCluster.java

```diff
@@ -66,8 +67,7 @@ public static void createCluster(String projectId, String region, String cluster
 
       // Print out a success message
       System.out.println(
-          String.format("Cluster created successfully: %s", response.getClusterName())
-      );
+          String.format("Cluster created successfully: %s", response.getClusterName()));
```

Comment:
This can be more simply done without String.format; just concatenate the string

Reply:
Personally I prefer `String.format` or `System.out.printf` over string concatenation, but either is preferable over nesting print/format.